config/obstacle_infos.yaml

```
@@ -0,0 +1,7 @@
+- obstacle1:
+    - type: "sphere"
+    - dim: 0.175
```

```suggestion
    - radious: 0.175
```
Please use the same naming as parameters in objects in hpp-fcl